eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -25,3 +25,4 @@ steps:
         -branchReplacementName ${{ parameters.BranchReplacementName }}
         -devOpsLogging: $true
         -checkLinkGuidance: ${{ parameters.CheckLinkGuidance }}
+    condition: succeededOrFailed()
```

Comment:
We should make this a template option as opposed to doing for everyone. Please add another template parameter and only add this if that someone opts into it.

Reply:
Do we want to make succeeded() as default?

Reply:
I'd just exclude it if the parameter isn't set.

Reply:
This is $true/false value, how do we exclude it if not set?

Reply:
```
 ${{ if eq(parameters.RunOnFailure, true) }}:
  condition: succeededOrFailed()
```

I believe something like that should work.